notebooks/py_scripts/05_Export.py

```diff
@@ -193,6 +193,58 @@
 # can use to replicate the database entries you used in your analysis.
 #
 
+# # Dandiset Upload
+
+# One benefit of the `Export` table is it provides a list of all raw data, intermediate analysis files, and final analysis files needed to generate a set of figures in a work. To aid in data-sharing standards, we have implemented tools to compile and upload this set of files as a Dandi dataset, which can then be used by spyglass to directly read the data from the Dandi database if not available locally.
+#
+# We will walk through the steps to do so here:
+
+# <details>
+#    <summary style="font-size:1.5em"> Dandi data compliance (admins)</summary>
+#
+#    >__WARNING__: The following describes spyglass utilities that require database admin privledges to run. It involves altering database values to correct for metadata format errors generated prior to spyglass insert. As such it has the potential to violate data integrity and should be used with caution.
+#    >
+#    >The Dandi database has specific formatting standards for metadata and nwb files. If there were violations of this standard in the
+#    raw nwbfile, spyglass will propagate them into all generated analysis files. In this case, running the code below will result in a list of error printouts and an error raised within the `validate_dandiset` function.
+#    >
+#    >To aid in correcting common formatting errors identified with changes in dandi standards, we have included the method
+#    ```
+#    Export()._prepare_files_for_export(paper_key)
```

Comment:
I was using this to try to indicate it's function that shouldn't be used frequently since it requires admin permissions.  Open to other ideas of how to indicate this or can remove

Reply:
I think it's sufficient for an unauthorized use to result in a permission error